EZEE-3255: Update Elasticsearch documentation in regard to self-signed CA certificates (#1146)
ezsystems/developer-documentation: docs/guide/search/elastic.md

    #### Configuring SSL

    When you need to protect your communication with the Elasticsearch server, you can use SSL encryption.

>     When configuring SSL for your internal infrastructure, you can use your own client certificates signed by public CA.

Reviewer: ```suggestion
    When configuring SSL for your internal infrastructure, you can use your own client certificates signed by a public CA.
```

                        path: '/path/to/cert-key'
                        pass: ~
    ```
    

>     It may occur that you don't posses client certificate signed by public CA authority
>     but you have self-signed CA certificate generated by `elasticsearch-certutil` tool 
>     or different (for example for developing purposes). In this case you should use the following `ssl` configuration:

Reviewer: ```suggestion
    If you do not have a client certificate signed by public certificate authority,
    but you have a self-signed CA certificate generated by `elasticsearch-certutil` or another tool (for example for development purposes),
    use the following `ssl` configuration:
```

    
    ``` yaml
    ezplatform_elastic_search_engine:
        connections:
            cloud_with_ssl:
                debug: true
                elastic_cloud_id: 	'test:ZWFzdHVzMi5henVyZS5lbGFzdGljLWNsb3VkLmNvbTo5MjQzJGUwZ'
                ssl:
                    ca_cert:
                        path: '/path/to/ca_cert.pem'
    ```
        

>     If you configure both `ca_cert` and `cert` entries the `ca_cert` parameter will take precedence over the `cert` parameter.

Reviewer: ```suggestion
    If you configure both `ca_cert` and `cert` entries, the `ca_cert` parameter takes precedence over the `cert` parameter.
```
